pyproject.toml

```
@@ -210,6 +210,7 @@ notebook = [
   'notebook~=6.1,>=6.1.5'
 ]
 pre-commit = [
+  'aiida-core[atomic_tools,pre-commit,rest,tests,tui]',
```

This might be too much recursion 😅 


```suggestion
  'aiida-core[atomic_tools,rest,tests,tui]',
```